### PR TITLE
CIV-8884 - CUI: create and link access roles 

### PR DESCRIPTION
**Pull request by neeta-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-8884

### Change description ###
added access roles.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by sankhajuria — APPROVED** [review]

File: ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles-CUI-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "CaseAccessCategories": ""
+  },
+  {
+    "RoleName" : "Defendant",
+    "Authorisation": "",
+    "CaseTypeID": "CIVIL",
+    "AccessProfiles": "DEFENDANT",
+    "ReadOnly": "N",
+    "Disabled": "F",
+    "CaseAccessCategories": ""
```
Comment (TatsyMaslova) [inline ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles-CUI-nonprod.json:18]: should not these have access to spec claim: `    "CaseAccessCategories": "SPEC_CLAIM"`

File: ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles-CUI-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "RoleName" : "Claimant",
+    "Authorisation": "",
+    "CaseTypeID": "CIVIL",
+    "AccessProfiles": "CLAIMANT",
+    "ReadOnly": "N",
+    "Disabled": "F",
+    "CaseAccessCategories": ""
```
Comment (TatsyMaslova) [inline ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles-CUI-nonprod.json:9]: should not these have access to spec claim

Comment (sampankumar) [reply ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles-CUI-nonprod.json:9]: Please change name for Access Profile
to something like
CITIZEN_DEFENDANT_PROFILE
CITIZEN_CLAIMANT_PROFILE

And yes CaseAccessCategories need to be SPEC_CLAIM for us

Comment (sampankumar) [reply ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles-CUI-nonprod.json:9]: Also role name is
"RoleName" : "[DEFENDANT]",
and 
"RoleName" : "[CLAIMANT]",

Comment (neeta-hmcts) [reply ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles-CUI-nonprod.json]: done

**Review by sankhajuria — APPROVED** [review]

**Pull request merged**